### PR TITLE
Show default value for arg in help message

### PR DESCRIPTION
**Pull request by cekc**

### CODE REVIEW

File: include/argparse/argparse.hpp [hunk 1]
```diff
@@ -73,6 +73,57 @@ struct is_container<
...
+    if (size > 0)
+      out << repr(*std::prev(val.end()));
+    out << "}";
+    return out.str();
+  } else if constexpr (is_streamable_v<T>) {
+    std::stringstream out;
+    out << val;
+    return out.str();
+  } else {
+    return "<not representable>";
```
Comment (p-ranav) [inline include/argparse/argparse.hpp:123]: Why not leave it empty if not representable?

**Review by p-ranav — APPROVED** [review]

**Review by p-ranav — CHANGES_REQUESTED** [review]
Please add some tests to validate this repr function

**Review by p-ranav — APPROVED** [review]

**Review by p-ranav — CHANGES_REQUESTED** [review]
Tests look good. Build seems to fail on Mac?

**Comment by cekc** [comment]
> Tests look good. Build seems to fail on Mac?

Yes, you see, similar build error happened earlier. Looks to me like the compiler sometimes renders trait `details::is_streamable_v<T>` to true incorrectly and thus breaks SFINAE.

**Review by p-ranav — APPROVED** [review]

**Comment by cekc** [comment]
Sorry for spamming with commits and builds. The problematic piece was
```cpp
decltype(std::declval<std::ostream&>() << std::declval<T>())
```
for trait `is_streamable`. I inserted that ampersand and all seem to work now. It's crazy that compilers behave differently.

Thank you for quick review and merge!

**Comment by p-ranav** [comment]
Thanks for the PR!

**Pull request merged**